code-challenges.test.js

```diff
@@ -15,45 +15,114 @@
+describe("divisibleBy3", () => {
+          var num1 = 15
+           var num2 = 0
+          var num3 = -7
+// I am describing the goal of the function which is to be able to take the numbers in the variable and check if the numbers are divisble by three
+
+    it("logs decides if the number is evenly divisble by three or not", () => {
+
+      expect(divisibleBy3(num1)).toEqual(`${number1} is divisible by three.`)
+      expect(divisibleBy3(num2)).toEqual(`${number2} is divisible by three.`)
+      expect(divisibleBy3(num3)).toEqual(`${number3} is not divisible by three.`)
+    })
+  })
+// I am expicting to get the result in each numbers of the variable.
+
+// // // b) Create the function that makes the test pass.
+// //
+const divisibleBy3 = (number1) => {
+  if ((number1)%3 === 0) {
+// I created a function by adding each numbers ( if the number is divisible by 3 the remaining will equal 0)
+    return `${number1} is divisible by three.`
+  } else {
+    return `${number1} is not divisible by three.`
+// because a function has to have a return I am telling what the return is expected
+  }
+}
+
+// var num1 = 15
+// // // Expected output: "15 is divisible by three"
+// var num2 = 0
+// // // Expected output: "0 is divisible by three"
+// var num3 = -7
+// // // Expected output: "-7 is not divisible by three"
 
+// --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
+// a) Create a test with expect statements for each of the variables provided.
+//
+describe("capitalizer", () => {
+// I am describing the goal of the test which is to have an array with all the words capitalized.
 
-// b) Create the function that makes the test pass.
+    var array1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+    var array2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
 
+    it("should an array with all the words capitalized", () => {
 
+// I am explaining that it should return an array of words capitalized
 
-// --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
+      expect(capitalizer(array1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+      expect(capitalizer(array2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango", "Deduction"])
+// the expected result
 
-// a) Create a test with expect statements for each of the variables provided.
+    })
+  })
+
+// // b) Create the function that makes the test pass.
+
+const capitalizer = (array) => {
+      let eachItem = array.map(value => {
+// I am creating a function that takes in the array and .map (will do something to each item in the array)
+
+      return value[0].toUpperCase() + value.substring(1)
+    })
+
+// adding the result will take the index of each word. toupercase will capitalized each first letter in the array
+    return eachItem
+  }
```

Comment:
The logic looks great. Just as a note, if you are creating a variable just to return it, you can skip it and just return the statement.